course/lesson-22-strings/lesson.tres

```diff
@@ -0,0 +1,198 @@
+[gd_resource type="Resource" load_steps=21 format=2]
+
+[ext_resource path="res://resources/Lesson.gd" type="Script" id=1]
+[ext_resource path="res://resources/QuizChoice.gd" type="Script" id=2]
+[ext_resource path="res://resources/ContentBlock.gd" type="Script" id=3]
+[ext_resource path="res://course/Documentation.tres" type="Resource" id=4]
+[ext_resource path="res://resources/Practice.gd" type="Script" id=5]
+
+[sub_resource type="Resource" id=1]
+script = ExtResource( 3 )
+content_id = "res://course/lesson-22-strings/content-jotxF5HU.tres"
+title = ""
+type = 0
+text = "Throughout this course, we've mostly stored numbers in variables. But what if we wanted to store a player's name? Using only numbers for names isn't very user-friendly at all, as 70801 here will tell you."
```

Comment:
Actually, this first example, which gives the robot a number as a name, uses a String. I'm unsure we should use that as a result, because as a dev you need to be really clear with types, their differences, and when you need to convert them (like here, you'd convert a number to a string to display it in a label)